tests/e2escenarios/e2e_scenarios_suite_test.go

```diff
@@ -1,13 +1,27 @@
 package e2escenarios
 
 import (
+	"fmt"
+	"os"
 	"testing"
+	"time"
 
 	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
 	. "github.com/onsi/gomega"
 )
 
 func TestE2eScenarios(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "odo e2e scenarios")
+	time := time.Now()
```

Comment:
The next 4 lines are repeated for each test suite, it would be nice to abstract it away to some nice function

Reply:
hmm, will fix it.
Thanks